src/main.js

```diff
@@ -105,14 +105,30 @@ var currentPoster;
 
 // functions and event handlers go here 👇
 // (we've provided two to get you started)!
-function getRandomIndex(array) {
-  return Math.floor(Math.random() * array.length);
+
+// function getRandomIndex(array) {
+//   return Math.floor(Math.random() * array.length);
+// }
+
+// function createPoster(imageURL, title, quote) {
+//   return {
+//     id: Date.now(), 
+//     imageURL: imageURL, 
+//     title: title, 
+//     quote: quote}
+// }
+
+function getRandomIndex(quotes) {
+  return Math.floor(Math.random() * quotes.length); 
 }
 
-function createPoster(imageURL, title, quote) {
-  return {
-    id: Date.now(), 
-    imageURL: imageURL, 
-    title: title, 
-    quote: quote}
-}
\ No newline at end of file
+var randomQuote = getRandomIndex(quotes)
+var posterQuote = document.querySelector('.poster-quote')
```

Comment:
I was meeting with some mod 2 folks, and they pointed out that all the query selectors go at the top, but I can merge and move it up there if you're ok with it!